### DISCUSSION TITLE
How to clear session from server-side

### DISCUSSION THREAD

**Discussion by rolanday** [nextauthjs/next-auth#6345] [Help]
### Question 💬

Does next-auth provide a mechanism for deleting session state from server-side, similar to calling signOut method in client?

My app is using JWTs for authenticating requests to the backend, which includes making calls to backend in next-auth signIn and jwt callbacks. I use the backend response data inside jwt callback to update token with session state, but if the request fails, then I want to tear-down the session inside the callback sequence so the client is not left in a bad state (i.e., logged in from next-auth perspective, but incomplete session state).

Any help much appreciated. Thanks!

### How to reproduce ☕️

N/A

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment]
You can follow #6337
